### DISCUSSION TITLE
App studenti

### DISCUSSION THREAD

**Discussion by luigiadoc** [iisgiua/giuaschool#801] [Richieste di aiuto] [answered]
Salve, esiste un'applicazione come quella per i docenti ("Giua Reg 3.0.") per gli studenti ?

**Comment by luigiadoc** [comment]
@trinko

**Comment by trinko** [comment]
L'app esistente GiuaReg (che a breve sarà aggiornata a una nuova versione con una nuova denominazione) è utilizzabile da qualsiasi tipologia di utente.

**Comment by luigiadoc** [comment]
Grazie. E' già possibile sapere cosa sarà aggiunto con la nuova versione dell'app ?
@trinko

**Reply by trinko** [reply, marked as answer]
> Grazie. E' già possibile sapere cosa sarà aggiunto con la nuova versione dell'app ? 

La nuova versione ha un nuovo sistema di login che semplifica l'accesso al registro, rendendolo indipendente dalle credenziali perché viene associato dispositivo usato all'utente sul registro.

La nuova versione dell'app diventerà l'unica utilizzabile con la versione del registro >= 1.6.0
